src/main/java/com/growup/pms/user/controller/UserControllerV1.java

```diff
@@ -63,4 +66,12 @@ public ResponseEntity<Void> sendVerificationCode(String email) {
         userService.sendVerificationCode(email);
         return ResponseEntity.ok().build();
     }
+
+    @PatchMapping("/password")
+    public ResponseEntity<Void> updatePassword(@AuthenticationPrincipal SecurityUser user,
```

Comment:
Nit: 이 부분은 하나의 애노테이션으로 통일해보면 좋을 것 같네요! :thinking:  
```suggestion
    public ResponseEntity<Void> updatePassword(@CurrentUser SecurityUser user,
```